trailblazer/store/api.py

```diff
@@ -200,32 +156,15 @@ def update_run_status(self, analysis_id: int, ssh: bool = False) -> None:
         if analysis.workflow_manager == WorkflowManager.TOWER.value:
             self.update_tower_run_status(analysis_id=analysis_id)
         elif analysis.workflow_manager == WorkflowManager.SLURM.value:
-            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id, ssh=ssh)
+            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id)
 
-    def update_analysis_from_slurm_run_status(self, analysis_id: int, ssh: bool = False) -> None:
-        """Query slurm for entries related to given analysis, and update the analysis in the database."""
-        analysis: Analysis = self.get_analysis_with_id(analysis_id=analysis_id)
+    def update_analysis_from_slurm_run_status(self, analysis_id: int) -> None:
```

Comment:
```suggestion
    def update_slurm_run_status(self, analysis_id: int) -> None:
```

Reply:
But we do not update the slurm run status (we do not control it). We update the analysis based on the current slurm run status.

Reply:
you are right. I do find the current name confusing though, the `run_status` part is misleading to me. But I don't have good alternatives to offer, `update_analysis_from_slurm_jobs` ?
 I leave it up to you, feel free to resolve

Reply:
👍